ClassTranscribeDatabase/CommonUtils.cs

```diff
@@ -19,7 +19,7 @@ public enum TaskType
             DownloadPlaylistInfo = 3,
             DownloadMedia = 4,
             ConvertMedia = 5,
-            Unused = 6,
```

Comment:
FYI TI changed my mind about removing TranscribeVideo task and creating two new tasks - and went back to the original TranscribeVideo task design